pkg/api/const.go

```diff
@@ -485,6 +485,8 @@ const (
 	DefaultNonMasqueradeCIDR = "0.0.0.0/0"
 	// DefaultKubeProxyMode is the default KubeProxyMode value
 	DefaultKubeProxyMode KubeProxyMode = KubeProxyModeIPTables
+	// DefaultWindowsSshEnabled is the default windowsProfile.sshEnabled value
+	DefaultWindowsSshEnabled = true
```

Comment:
nit: prefer `DefaultWindowsSSHEnabled` (SSH instead of Ssh)

Reply:
Yea... this failed linting.
Cleaning this up now.